### ISSUE TITLE
Query Builder didn't show up

### ISSUE THREAD

**Issue by TheMapSmith** [raykendo/Map-Services-Enhanced#7]
Hello,

I was very interested in the tool to use the interactive query builder, but it did not show up on the page. This error shows up in the console:

![image](https://user-images.githubusercontent.com/8357118/28173071-582f2fc4-67bb-11e7-8411-e79c6e78f7b3.png)

**Comment by raykendo** [comment]
Interesting. I'll take a look and see where that is coming from. Is the map or feature service used by the query builder publicly available?

**Comment by TheMapSmith** [comment]
It's a feature service on MS SQL 2016, SDE 10.4.1, Server 10.4.1 on IIS. I'll send you the url separately.

**Comment by raykendo** [comment]
After investigating the url, I learned that feature services have a query page just off of their main feature service page, and not just at every level. After implementing the Find Task Helper, I was able to implement a version of the query helper that works off of the feature service main page. After looking at the page, however, more work will need to be done to make it more relevant.

At the very least, the error stops, and the widget should load.
